packages/blockchain-link/src/workers/solana/fee.ts

```diff
@@ -0,0 +1,37 @@
+import { Connection, Message, PublicKey } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+
+const DEFAULT_COMPUTE_UNIT_PRICE = 100_000; // micro-lamports, value taken from other wallets
+const DEFAULT_COMPUTE_UNIT_LIMIT = 50_000; // sending tokens with token account creation requires ~28K units
+
+export const getBaseFee = async (api: Connection, message: string) => {
+    const result = await api.getFeeForMessage(Message.from(Buffer.from(message, 'hex')));
+    // The result can be null, for example if the transaction blockhash is invalid.
+    // In this case, we should fall back to the default fee.
+    if (result.value == null) {
+        throw new Error('Could not estimate fee for transaction.');
+    }
```

Comment:
Since this throws when the value is null, the priority fee will never get calculated in `blockchainEstimateFee`. Do you think that should happen or do we want to add a default priority fee on top in that case too?

Reply:
There is a [default priority fee configured](https://github.com/vacuumlabs/trezor-suite/pull/34/commits/33c5bc78b9c899bf19b14a51f140461af0689507) which will be used if this throws. I'm not sure exactly how it works in the background but it does.